docs/sources/api/_index.md

```diff
@@ -1114,8 +1114,17 @@ Deletes all the rule groups in a namespace (including the namespace itself). Thi
 
 _This experimental endpoint is disabled by default and can be enabled via the `-experimental.ruler.enable-api` CLI flag (or its respective YAML config option)._
 
-_Requires [authentication](#authentication)._
+### List rules
 
+```
+GET /loki/api/v1/rules
+```
+
+Prometheus-compatible rules endpoint to list alerting and recording rules that are currently loaded.
+
+_For more information, please check out the Prometheus [rules](https://prometheus.io/docs/prometheus/latest/querying/api/#rules) documentation._
+
+_This experimental endpoint is disabled by default and can be enabled via the `-experimental.ruler.enable-api` CLI flag (or its respective YAML config option)._
```

Comment:
Seems like you should lead with this. "It doesn't work unless you do this specific thing" is kind of important.